samples/azure-quantum/resource-estimation/estimation-dynamics.ipynb

```diff
@@ -380,6 +495,63 @@
     "All other parameters remain unchanged."
```

Comment:
yes this is a good point, i wanted to include the time and space diagrams in the results section but wasnt sure what to do with this section. I moved it to be under the time chart. Let me know what you think about that instead. It could also go right below the results summary table and then we have the space and time diagrams.